CHANGELOG.md

```diff
@@ -1,4 +1,12 @@
+## 1.5.1
```

Comment:
I'm good with a z. Strictly speaking it's an additive backwards compatible thing, but eh